src/components/hero/campaignHeroSubText.tsx

```diff
@@ -35,9 +38,17 @@ const CampaignHeroSubText = ({ featuredItem }: any) => {
           }}
         >
           Featured Image:{" "}
-          <DSLink hasVisitedState={false} href={featuredItem.href}>
-            {featuredItem.title}{" "}
-          </DSLink>
+          {isLargerThanXLarge ? (
+            <Tooltip content={featuredItem.title}>
+              <DSLink hasVisitedState={false} href={featuredItem.href}>
+                {featuredItem.title}{" "}
+              </DSLink>
```

Comment:
Minor: define the link component as a variable. If you need to make an update to the link, you'll have to make it in two places.